server/plugin.go

```diff
@@ -142,25 +144,33 @@ func (p *Plugin) updateOrAppend(originals []analyticsData, upsert analyticsData)
 	return append(originals, upsert)
 }
 
-func (p *Plugin) getChannelName(key string) (string, string, *model.AppError) {
+func (p *Plugin) getChannelName(key string) (string, string, string, *model.AppError) {
```

Comment:
Would you please add a GoDoc comment to this method? It's not clear what the three return parameters are.

Reply:
:heavy_check_mark: